docker/quickstart/docker-compose.yml

```diff
@@ -234,7 +234,7 @@ services:
       - kafka-setup
       - elasticsearch
       - neo4j
-    command: "sh -c 'while ping -c1 kafka-setup &>/dev/null; do echo waiting for kafka-setup... && sleep 1; done; \
```

Comment:
With the latest `datahub-mce-conusmer` & `datahub-mae-consumer`, changing `sh` back to `bash` actually leads the the following error. Could you verify form a clean setup?

```
ERROR: for datahub-mae-consumer  Cannot start service datahub-mae-consumer: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"bash\": executable file not found in $PATH": unknown
```

Reply:
Reverted the changes.